utils/p4runtime_lib/error_utils.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, message):
 # index of the operation in the batch that failed and the second element being
 # the p4.Error Protobuf message.
 def parseGrpcErrorBinaryDetails(grpc_error):
+    # Check if grpc_error is None or not an instance of grpc.RpcError
+    if not grpc_error or not isinstance(grpc_error, grpc.RpcError):
+        raise P4RuntimeErrorFormatException("Invalid gRPC error object")
+        
```

Comment:
As a developer that might be calling code where this new `if` statement condition is true, note that the new exception raised is leaving out details that caused the exception.  In particular, it leaves out the value of `grpc_error`.  I am not a Python expert, but is there a "Pythonic" way to include the value of `grpc_error` as a field of the exception being raised, so that a caller could extract it out, or at least print it and see its value, if this new `if` condition is true?

Reply:
I've used an `f-string` (formatted string literal) to include the value of `grpc_error` in the exception message. This way, when the exception is raised, it will provide more context by including the actual value of `grpc_error`.